portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,56 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+let i = 0;
+let facts = [
+  "Digital Arts and Sciences Engineer ",
+  "Full-Stack Developer ",
+  "Scrum Master ",
+  "Google SPS participant! >:) "
+];
+let txt = "I'm a " + facts[0];
+let speed = 50;
+let backspace = false;
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+window.onload = function() {
+  typeWriter();
+  i = 0;
+  txt = "I'm a " + facts[0];
+  backspace = false;
+};
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function typeButton() {
+  typeWriter();
+  i = 0;
+  txt = "I'm a " + facts[Math.floor(Math.random() * facts.length)];
+  backspace = false;
+}
+
+function typeWriter() {
```

Comment:
I would consider renaming this function to make it more obvious what it does? Something like `addCharacterToDisplayedMessage()` maybe?

Reply:
That makes sense, but addCharacterToDisplayedMessage seems kind of long, would typeFact() be a little more descriptive? Since that's technically what it's supposed to do

Reply:
I personally don't mind long function or variable names, but if you want to go with `typeFact()`, maybe it would also help to add a comment above the function explaining that it adds a single character to the displayed message?

Reply:
Okay, I'm going to do that!